main.go

```diff
@@ -63,13 +62,7 @@ func handleJobs(jobsC <-chan context.Job) {
 func executeTasks(tasks []*context.Task, watcher *context.Watcher) {
 	for _, task := range tasks {
 		command := task.Command
-		tokens := strings.Split(command, " ")
-		name := tokens[0]
-		var cmdArg []string
-		if len(tokens) > 1 {
-			cmdArg = tokens[1:]
-		}
-		cmd := exec.Command(name, cmdArg...)
+		cmd := exec.Command("/bin/bash", "-c", command)
```

Comment:
> its not just env variables, it could be another command as well, which could have a env variable inside....

I'm not sure I'm following. Can you illustrate your words by an example of such command?